osu.Game/Online/Chat/NowPlayingCommand.cs

```diff
@@ -59,9 +67,31 @@ protected override void LoadComplete()
                     break;
             }
 
-            string beatmapString = beatmapInfo.OnlineID > 0 ? $"[{api.WebsiteRootUrl}/b/{beatmapInfo.OnlineID} {beatmapInfo}]" : beatmapInfo.ToString();
+            string beatmapString()
+            {
+                string beatmapInfoString = localisation.GetLocalisedBindableString(beatmapInfo.GetDisplayTitleRomanisable()).Value;
+
+                return beatmapInfo.OnlineID > 0 ? $"[{api.WebsiteRootUrl}/b/{beatmapInfo.OnlineID} {beatmapInfoString}]" : beatmapInfoString;
+            }
+
+            string modString()
+            {
+                if (selectedMods.Value.Count == 0)
+                {
+                    return string.Empty;
+                }
+
+                StringBuilder modS = new StringBuilder();
+
+                foreach (var mod in selectedMods.Value)
+                {
+                    modS.Append($"+{mod.Name} ");
+                }
```

Comment:
Two concerns:

- Where's this `+{mod name}` format even coming from? Is this standardised anywhere?
- Why use full name and not acronym? This is especially salient when you note that chat messages have a limit of 100 chars.

Reply:
> * Where's this `+{mod name}` format even coming from? Is this standardised anywhere?

I assume it's taken from stable /np command
![image](https://user-images.githubusercontent.com/50285552/204084868-18a034fa-27a9-4be6-b16d-b7265076f675.png)

And as you can see, stable uses different marks for difficulty increase/decrease/Special mods, not just a `+ `.  And stable also uses full mod name, but I agree it takes a lot of space since lazer has way more mods than stable